### DISCUSSION TITLE
Settings Filter Bar

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#732] [Show and tell]
A small thing, but this will hopefully help the UX in the Settings and Configuration editors:

![tbSettingsFilter](https://user-images.githubusercontent.com/25738109/153701217-89e834ee-3f1e-4256-9d6d-9a9d83a22407.gif)

**_Feature available in v0.14.18_**

**Comment by mwolfe02** [comment]
Very nice.  I noticed it doesn't currently search Compiler Warning descriptions (e.g., `DefType` returns no results).  

As the list of Compiler Warnings grows, I think that would be a nice added feature, though I'm not sure if that would be better included as a dedicated "Compiler Warning Search."  There are only 8 compiler warnings currently, so it's not something to address any time soon, but I thought I'd get it on the radar.

**Reply by Kr00l** [reply]
When there are 100 compiler warnings (hopefully not!) someday it would  make sense to have a ADODC-like Navigation-Bar which displays 10 warnings per page. ;) thus avoiding a bloated settings page.
Of course that could be in addition of the search which would then reduce the 100 warnings to maybe 5 which then would fit into 1 page.
